### DISCUSSION TITLE
简化aave protocol

### DISCUSSION THREAD

**Discussion by guotie** [rebase-network/hello-world#34] [Ideas]
打算简化 aave protocol，并增加以下功能：
1. permissionless lending
2. 算法稳定币

**Comment by luckyyang** [comment]
赞，基于 lending pool 的算法稳定币吗

**Reply by guotie** [reply]
打算类似于frax的稳定机制，但是基于ETH发行

**Reply by luckyyang** [reply]
不会对借贷造成影响吗

**Reply by guotie** [reply]
对借贷没有影响，但是可能发生eth大幅下跌时，没有足够的eth兑付

**Reply by luckyyang** [reply]
了解，如果有更多的介绍和设计可以分享一下
